doc/api/buffer.md

```diff
@@ -404,14 +404,14 @@ to initialize a `Buffer` to zeroes.
 Example:
 
 ```js
-const buf = new Buffer(5);
+const buf = new Buffer(10);
```

Comment:
```
core/tls-node (notes-tls $% u=) % node f.js
<Buffer 48 06 65 02 00>
<Buffer 00 00 00 00 00>
core/tls-node (notes-tls $% u=) % cat f.js
const buf = new Buffer(5);
// Prints: (contents may vary): <Buffer 78 e0 82 02 01>
console.log(buf);
buf.fill(0);

// Prints: <Buffer 00 00 00 00 00>
console.log(buf);
```

Contents may vary mean it can be anything, including zeroed. Maybe just add a comment to that effect? Looks like your system might always zero buffers 8 bytes or smaller?